packages/sdk/src/libs/clients/registry/registry.ts

```diff
@@ -0,0 +1,55 @@
+import assert from "assert";
+import {Registry__factory} from "@uma/core/contract-types/ethers"
+import Registry from "@uma/core/build/contracts/Registry.json"
+import type { SignerOrProvider} from '../../index.d'
+import {EventFragment, Result} from '@ethersproject/abi'
+import { BlockTag} from "@ethersproject/abstract-provider";
+import {EventFilter,Event} from 'ethers'
+
+export interface TypedEvent<EventArgs extends Result> extends Event {
+  args: EventArgs;
+}
+type NewContractRegisteredEvent = TypedEvent< [string, string, string[]] & { contractAddress: string; creator: string; parties: string[] } >;  
```

Comment:
this is copied from typechain outputs. I dont know how to actually import this, it would be really nice